eclipse-theia/theia
# Update "Open With..." command UX

#### What it does

Updates our UX for the "Open With..." menu entry to align with vscode

Previously:

![image](https://github.com/eclipse-theia/theia/assets/4377073/09f0ad9b-a649-44c8-a256-14263005d688)

After:

![image](https://github.com/eclipse-theia/theia/assets/4377073/224da760-59fb-4ef4-861f-fd77f4ef3084)

#### How to test

Notebook editors should work as expected with this:

1. Add a notebook file (`.ipynb`) to your workspace
2. Run the "Open With..." command via the context menu for that file
3. It should show the text editor and Jupyter notebook options

Custom editors should still work as expected with this:

1. Install the `Hex Editor` extension
2. Run the "Open With..." command via the context menu for any file
3. It should show the text editor and Hex editor options (among others)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
